blockly-rtc/typedefs.js

```diff
@@ -37,6 +37,16 @@
  * @property {string} entryId The id assigned to an event by the client.
  */
  
+ /**
+ * The location of a Marker.
+ * @typedef {Object} MarkerLocation
+ * @property {<string>} type The type of element.
```

Comment:
I don't think you need the angle brackets for strings.

Reply:
Done.